synapse/lib/time.py

```diff
@@ -245,3 +245,13 @@ def delta(text):
         raise s_exc.BadTypeValu(name='time', valu=text, mesg=mesg)
 
     return size * base
+
+def toutc(tick, fromzone):
+    try:
+        tz = pytz.timezone(fromzone)
+    except pytz.exceptions.UnknownTimeZoneError:
+        mesg = f'Unknown timezone: {fromzone}'
+        raise s_exc.BadArg(mesg=mesg)
```

Comment:
we should either use exception chaining here or break the chain
```
    try:
        tz = pytz.timezone(fromzone)
    except pytz.exceptions.UnknownTimeZoneError as e:
        mesg = f'Unknown timezone: {fromzone}'
        raise s_exc.BadArg(mesg=mesg) from e
```

`        raise s_exc.BadArg(mesg=mesg, fromzone=fromzone) from None`